server/vulnerabilities/nvd/cve.go

```diff
@@ -588,6 +588,14 @@ func expandCPEAliases(cpeItem *wfn.Attributes) []*wfn.Attributes {
 		}
 	}
 
+	for _, cpeItem := range cpeItems {
+		if cpeItem.Vendor == "oracle" && cpeItem.Product == "virtualbox" {
+			cpeItem2 := *cpeItem
+			cpeItem2.Product = "vm_virtualbox"
+			cpeItems = append(cpeItems, &cpeItem2)
+		}
+	}
+
```

Comment:
Why not do this in `cpe_translations.json`?

Reply:
`virtualbox` is a valid product name for some CVEs and adding this rule to cpe_translations will overwrite the productname rather than append an additional search for `vm_virtualbox`.  I think a worthy longer term investment to add this append pattern into cpe_translations.